src/extension.ts

```diff
@@ -56,6 +56,7 @@ const DMI_PRODUCT_VERSION_PATH = "/sys/class/dmi/id/product_version";
 const DISCRETE_EXTERNAL_DISPLAY_MODELS = [
     "addw1",
     "addw2",
+	"gaze17-3060-b",
```

Comment:
A tab? :vomiting_face:

Reply:
It never happened. Also, still looking forward to when I can configure my vim to not fuck up whitespace in every project